Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "MCMCDiagnosticTools"
 uuid = "be115224-59cd-429b-ad48-344e309966f0"
 authors = ["David Widmann"]
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
Not allowed to bump Julia version in patch releases.

Reply:
Is this documented somewhere? I didn't see this in https://pkgdocs.julialang.org/v1/compatibility/. ColPrac makes a good argument though for incrementing the minor version, but it's not required: https://github.com/SciML/ColPrac#dropping-support-for-earlier-versions-of-julia

It doesn't seem to be enforced, since I recently released a patch that did this: https://github.com/JuliaRegistries/General/pull/56946

Reply:
It's documented here: https://juliaregistries.github.io/RegistryCI.jl/stable/guidelines/#New-versions-of-existing-packages